starwars/src/components/CharContainer.js

```diff
@@ -0,0 +1,62 @@
+import React, {useState, useEffect} from 'react';
+import styled from 'styled-components';
+import CharCard from './CharCard';
+import axios from 'axios';
+
+
+const Container = styled.div`
+    border:2px solid black;
+    width: 80%;
+    margin: 0 auto;
+    display:flex;
+    flex-wrap:wrap;
+    justify-content:center;
+    justify-content: space-evenly;
+    padding:3%;
+    background: rgba(0, 0, 0, 0.5);
+    margin-bottom: 5%;
+`;
+
+
+
+const CharContainer = () => {
+
+    const [starChars, setStarChars] = useState([]);
+
+
+    useEffect(() => {
+        axios.get('https://swapi.co/api/people/')
```

Comment:
do this up in the APP and pass into your bridge between app and individual cards

Reply:
D'oh! I initially had state/side-effect/get request in App.js, and moved it to CharContainer.js... and now I can't even remember why!